### PR TITLE
changes for part 2

### PR DESCRIPTION
**Pull request by joshmckibben**

### CODE REVIEW

**Comment by joshmckibben** [comment]
part 2 of case study.

File: activities.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+</head>
+
+<body>
+
+   <div id="wrapper">
+
+      <h1>Pacific Trails Resort</h1>
+
+      <div id="nav"><a href="index.html">Home</a> &nbsp;
```
Comment (mbMosman) [inline activities.html]: These should be HTML 5 pages, so use the `<nav>` tag, not a div with the id of nav.

File: activities.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+      <h1>Pacific Trails Resort</h1>
+
+      <div id="nav"><a href="index.html">Home</a> &nbsp; 
+                    <a href="yurts.html">Yurts</a> &nbsp; 
+                    <a href="activities.html">Activites</a> &nbsp; 
+                    <a href="reservations.html">Reservations</a>
+      </div>
+
+      <div id="content">
```
Comment (mbMosman) [inline activities.html]: This should be a `<main>` tag.

File: activities.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+         <p>Ocean kayaks are available for guest use.</p>  
+
+         <h3>Bird Watching</h3>
+
+         <p>While anytime is a good time for bird watching at Pacific Trails, 
+            we offer guided birdwatching trips at sunrise several times a week.</p>
+
+      </div>
+
+      <div  id="footer"> Copyright &copy; 2015 Pacific Trails Resort<br>
```
Comment (mbMosman) [inline activities.html]: These should be HTML 5 pages, so use the `<footer>` tag, not a div with the id of footer.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+</head>
+
+<body>
+
+   <div id="wrapper">
+
+      <h1>Pacific Trails Resort</h1>
+
+      <div id="nav"><a href="index.html">Home</a> &nbsp;
```
Comment (mbMosman) [inline index.html]: These should be HTML 5 pages, so use the `<nav>` tag, not a div with the id of nav.

File: yurts.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+</head>
+
+<body>
+
+   <div id="wrapper">
+
+      <h1>Pacific Trails Resort</h1>
+
+      <div id="nav"><a href="index.html">Home</a> &nbsp;
```
Comment (mbMosman) [inline yurts.html]: These should be HTML 5 pages, so use the `<nav>` tag, not a div with the id of nav.

File: yurts.html [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+                   Shower and restroom facilities are located in the lodge.<br><br></dd>
+
+            <dt><strong><b>What should I bring?</b></strong></dt>
+               <dd>Bring a sense of adventure and some time to relax! Most guests also pack comfortable 
+                   walking shoes and plan to dress for changing weather with layers of clothing.</dd>
+         </dl>
+
+      </div>
+
+      <div  id="footer"> Copyright &copy; 2015 Pacific Trails Resort<br>
```
Comment (mbMosman) [inline yurts.html]: These should be HTML 5 pages, so use the `<footer>` tag, not a div with the id of footer.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+            12010 Pacific Trails Road<br>
+            Zephyr, CA 95555<br>
+            <br>
+            888-555-5555<br>
+            <br>
+         </div>
+
+      </div>
+
+      <div  id="footer"> Copyright &copy; 2015 Pacific Trails Resort<br>
```
Comment (mbMosman) [inline index.html]: These should be HTML 5 pages, so use the `<footer>` tag, not a div with the id of footer.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+      <h1>Pacific Trails Resort</h1>
+
+      <div id="nav"><a href="index.html">Home</a> &nbsp; 
+                    <a href="yurts.html">Yurts</a> &nbsp; 
+                    <a href="activities.html">Activites</a> &nbsp; 
+                    <a href="reservations.html">Reservations</a>
+      </div>
+
+      <div id="content">
```
Comment (mbMosman) [inline index.html]: This should be a `<main>` tag.

File: yurts.html [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+      <h1>Pacific Trails Resort</h1>
+
+      <div id="nav"><a href="index.html">Home</a> &nbsp; 
+                    <a href="yurts.html">Yurts</a> &nbsp; 
+                    <a href="activities.html">Activites</a> &nbsp; 
+                    <a href="reservations.html">Reservations</a>
+      </div>
+
+      <div id="content">
```
Comment (mbMosman) [inline yurts.html]: This should be a `<main>` tag.

File: yurts.html [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+
+         <h2>The Yurts at Pacific Trails Resort</h2>
+
+         <img src="images\yurt.jpg" alt="Yurt" height="250" width="320">
+
+         <dl>
```
Comment (mbMosman) [inline yurts.html]: Please be sure that you are using the instructions from the course web site. They specify this should be setup as `<h3>` tags and `<p>` tags.

File: pacific.css [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+h1 { background-color: #000033; 
+     background-image: url(images/sunset.jpg); 
+     background-position: right; 
+     background-repeat: no-repeat; 
+     padding-left: 20px; 
+     height: 72px; 
+     color: #FFFFFF; 
+     font-family: Georgia, serif; 
+     line-height: 200%;
```
Comment (mbMosman) [inline pacific.css]: There is no need to set the height or the line-height here.

File: pacific.css [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+h1 { background-color: #000033; 
+     background-image: url(images/sunset.jpg); 
+     background-position: right; 
+     background-repeat: no-repeat; 
+     padding-left: 20px; 
+     height: 72px; 
+     color: #FFFFFF; 
+     font-family: Georgia, serif;
```
Comment (mbMosman) [inline pacific.css]: The correct font-family with fall backs should be:` font-family: Georgia, Times, "Times New Roman", serif`

File: pacific.css [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+h2 { color: #3399CC; 
+     font-family: Georgia, serif; 
+}
+
+h3 { color: #000033; 
+}
+
+dt { color: #000033;
```
Comment (mbMosman) [inline pacific.css]: This is not needed as the Yurts page should use `<h3>` and `<p>` tags.

File: pacific.css [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+     padding-left: 20px; 
+     height: 72px; 
+     color: #FFFFFF; 
+     font-family: Georgia, serif; 
+     line-height: 200%; 
+     margin-bottom: 0; 
+}
+
+h2 { color: #3399CC; 
+     font-family: Georgia, serif;
```
Comment (mbMosman) [inline pacific.css]: Use the CSS font stack site or a similar reference to get fall back fonts.

File: pacific.css [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+     font-family: Georgia, serif; 
+     line-height: 200%; 
+     margin-bottom: 0; 
+}
+
+h2 { color: #3399CC; 
+     font-family: Georgia, serif; 
+}
+
+h3 { color: #000033;
```
Comment (mbMosman) [inline pacific.css]: The h3 headings should also use the Georgia fonts.

File: pacific.css [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+#nav { background-color: #90C7E3; 
+       font-weight: bold; 
+       padding: 5px; 
+}
+
+/* Configure Footer */
+#footer { font-size: .70em; 
+          font-style: italic; 
+          text-align: center; 
+          padding: 10px;
```
Comment (mbMosman) [inline pacific.css]: The footer should also use the Georgia fonts.

File: pacific.css [hunk 17]
```diff
@@ -0,0 +1,64 @@
...
+/* Configure Content */
+#content { padding-left: 20px; 
+           padding-right: 20px; 
+}
+
+/* Configure Navigation */
+#nav { background-color: #90C7E3; 
+       font-weight: bold; 
+       padding: 5px; 
+}
```
Comment (mbMosman) [inline pacific.css]: The underline should be removed from the nav links.  This requires a descendant selector to target the `<a>` tags.

File: pacific.css [hunk 18]
```diff
@@ -0,0 +1,64 @@
...
+
+h3 { color: #000033; 
+}
+
+dt { color: #000033; 
+}
+
+/* Configure Resort Name */
+.resort { color: #000033; 
+          font-size: 1.2em;
```
Comment (mbMosman) [inline pacific.css]: This should not have a larger font, but should be bold.

File: index.html [hunk 19]
```diff
@@ -0,0 +1,64 @@
...
+         <h2>Enjoy Nature in Luxury</h2>
+
+         <img src="images\coast.jpg" alt="Coast" height="250" width="320">
+
+         <p><span class="resort">Pacific Trails Resort</span> offers a special lodging experience on the Califronia North Coast. 
+            Relax in serenity with panoramic views of the Pacific Ocean.
+         </p>
+
+         <ul>
+            <li>Private yurts with decks overlooking the ocean</li>
```
Comment (mbMosman) [inline index.html]: CSS should be configured to show these using the marker.gif.

File: index.html [hunk 20]
```diff
@@ -0,0 +1,64 @@
...
+            <li>Activities lodge with fireplace and gift shop</li>
+            <li>Nightly fine dining at the Overlook Cafe</li>
+            <li>Heated outdoor pool and whirlpool</li>
+            <li>Guided hiking tours of the redwoods</li>
+         </uL>
+
+         <div>
```
Comment (mbMosman) [inline index.html]: This div should be configured with an id="contact" which is used to reduce the font size to 90%.

File: index.html [hunk 21]
```diff
@@ -0,0 +1,64 @@
...
+            Pacific Trails Resort<br>
```
Comment (mbMosman) [inline index.html]: This should also have the resort style class applied.

File: pacific.css [hunk 22]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+/* Configure Resort Name */
+.resort { color: #000033; 
+          font-size: 1.2em; 
+}
+
+/* Configure Content */
+#content { padding-left: 20px; 
+           padding-right: 20px;
```
Comment (mbMosman) [inline pacific.css]: padding should be 1px on the top and 20px on all other sides.

File: pacific.css [hunk 23]
```diff
@@ -0,0 +1,64 @@
...
+          font-size: 1.2em; 
+}
+
+/* Configure Content */
+#content { padding-left: 20px; 
+           padding-right: 20px; 
+}
+
+/* Configure Navigation */
+#nav { background-color: #90C7E3;
```
Comment (mbMosman) [inline pacific.css]: The nav text should be centered.

File: pacific.css [hunk 24]
```diff
@@ -0,0 +1,64 @@
+/* filename: pacific.css */
+/* coded by: Josh McKibben */
+
+body { background-color: #FFFFFF; 
+       background-image: url(images/ptrbackground.jpg); 
+       color: #666666; 
+       font-family: Verdana, Arial, sans-serif; 
+}
+
+h1 { background-color: #000033;
```
Comment (mbMosman) [inline pacific.css]: The header/h1 text should be centered.

File: pacific.css [hunk 25]
```diff
@@ -0,0 +1,64 @@
+/* filename: pacific.css */
+/* coded by: Josh McKibben */
+
+body { background-color: #FFFFFF; 
+       background-image: url(images/ptrbackground.jpg); 
+       color: #666666; 
+       font-family: Verdana, Arial, sans-serif;
```
Comment (mbMosman) [inline pacific.css]: The default font should be Arial.  The correct font-family would be `font-family: Arial, "Helvetica Neue", Helvetica, sans-serif`

File: yurts.html [hunk 26]
```diff
@@ -0,0 +1,57 @@
...
+                    <a href="yurts.html">Yurts</a> &nbsp; 
+                    <a href="activities.html">Activites</a> &nbsp; 
+                    <a href="reservations.html">Reservations</a>
+      </div>
+
+      <div id="content">
+
+         <h2>The Yurts at Pacific Trails Resort</h2>
+
+         <img src="images\yurt.jpg" alt="Yurt" height="250" width="320">
```
Comment (mbMosman) [inline yurts.html]: The `/` must be used as the path delimiter on the web.  This does not validate.  Be sure to run your code through the W3C HTML Validator before submitting the assignments.

File: index.html [hunk 27]
```diff
@@ -0,0 +1,64 @@
...
+                    <a href="yurts.html">Yurts</a> &nbsp; 
+                    <a href="activities.html">Activites</a> &nbsp; 
+                    <a href="reservations.html">Reservations</a>
+      </div>
+
+      <div id="content">
+
+         <h2>Enjoy Nature in Luxury</h2>
+
+         <img src="images\coast.jpg" alt="Coast" height="250" width="320">
```
Comment (mbMosman) [inline index.html]: The `/` must be used as the path delimiter on the web.  This does not validate.  Be sure to run your code through the W3C HTML Validator before submitting the assignments.

File: activities.html [hunk 28]
```diff
@@ -0,0 +1,56 @@
...
+                    <a href="yurts.html">Yurts</a> &nbsp; 
+                    <a href="activities.html">Activites</a> &nbsp; 
+                    <a href="reservations.html">Reservations</a>
+      </div>
+
+      <div id="content">
+
+         <h2>Activities at Pacific Trails Resort</h2>
+
+         <img src="images\trail.jpg" alt="Trail" height="250" width="320">
```
Comment (mbMosman) [inline activities.html]: The `/` must be used as the path delimiter on the web.  This does not validate.  Be sure to run your code through the W3C HTML Validator before submitting the assignments.

**Comment by mbMosman** [comment]
Overall this needs some cleanup, but is on the right path.  Be sure to run the the files through the W3C validators before submission.  I am letting it slide this time, but any validation error is generally a 10pt reduction, which drops the score a letter grade.

**Pull request closed without merging**